.circleci/config.yml

```diff
@@ -0,0 +1,87 @@
+version: 2
+jobs:
+
+  test:
+    docker:
+      - image: rust:1.31
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-cargo-cache-{{ arch }}-{{ .Branch }}
+            - v1-cargo-cache-{{ arch }}
+      - run:
+          name: Show versions
+          command: rustc --version && cargo --version
+      - run:
+          name: Build
+          command: cargo build
```

Comment:
Should we also use `-D warnings` here? As long as we don't bump the Rust version it won't break.

Reply:
On the other hand maybe we shouldn't do that, since third party code may trigger warnings.